components/AddEventForm/index.js

```diff
@@ -44,68 +122,45 @@ export default function AddEventForm({ onSubmit, appointment }) {
           <option value="wishlist">Wishlist</option>
           <option value="latest">Latest</option>
           <option value="upcoming">Upcoming</option>
-        </select>
-      </label>
-
-      <label>
-        Title:
-        <input
-          required
-          maxLength={20}
-          type="text"
-          name="title"
-          defaultValue={appointment ? appointment.title : null}
-        ></input>
-      </label>
-      <label>
-        Description:
-        <textarea
-          required
-          maxLength={300}
-          rows={10}
-          type="text"
-          name="description"
-          defaultValue={appointment ? appointment.description : null}
-        ></textarea>
-      </label>
-      <label>
-        Cost:
-        <input
+        </StyledSelect>
+        <StyledLabel htmlFor="cost">Cost in €: </StyledLabel>
+        <StyledInputCost
+          id="cost"
           required
           type="number"
           min={0}
           max={999999}
           name="cost"
           defaultValue={appointment ? appointment.cost : null}
-        ></input>
-        €
-      </label>
-      <label>
-        Priority:
-        <select
+        ></StyledInputCost>
```

Comment:
inputs are self-closing:

```suggestion
        />
```

Reply:
Done